docs/v3/group-chat.md

```diff
@@ -0,0 +1,165 @@
+
+## Conversations Protocol
+
+The conversations protocol contains methods to create groups, and monitor all groups the installation is part of. Each SDK has their own version of this protocol, using the underlying implementation in [LibXMTP](https://github.com/xmtp/libxmtp).
+
+| Method | Description |
+| ---- | ---- |
+| create_group | Create a new group chat for multiple account addresses. Syncs automatically.|
+| sync | Sync for welcome messages, creating new groups if one is found. |
+| list | Returns a list of groups matching the chosen time frame and limit. Requires sync. |
+| stream | Streams messages from the current group id to a callback function. |
+| stream_all_messages | Streams all messages to a callback function. |
+
+### Local Database and Syncing
+
+:::info More Information
+You can find more information on database operations in [XMTP MLS](https://github.com/xmtp/libxmtp/blob/main/xmtp_mls/README.md).
+:::
+
+When an app first calls `Client.create()` in one of the XMTP APIs, LibXMTP creates a local database to manage messaging between the app and the network. In subsequent calls, it loads the existing database. The database is encrypted using the keys from the `Signer` interface. See [Authentication](https://xmtp.org/docs/build/authentication#saving-keys) for instructions on how to extract the key, store it, and pass it to `Client.create()` later.  If you want to delete your local database, call `Client.deleteLocalDatabase()`.
+
+Calling the `sync()` method on the Conversations protocol checks for welcome messages and creates a new group if one is found. The new group will not show up on the list until the app syncs. Calling the `sync()` method on the Group protocol prompts LibXMTP to pull updates from the network, and push any unsent updates from the client related to the group. This will update the database for the group.
+
+In some cases, LibXMTP will perform a sync automatically. This currently includes calls to add/remove members to a group or using the stream functions. Outside of those methods, apps can call `sync()` when they choose in order to efficiently manage network traffic.
+
+## Group Protocol
+
+The group protocol contains methods specific to individual groups, identified by their `group_id`. These methods allow a user to communicate with the group. Each SDK has their own version of this protocol, using the underlying implementation in [LibXMTP](https://github.com/xmtp/libxmtp).
+
+| Method | Description |
+| ---- | ---- |
+| send | Send a message to this group. Requires sync. |
+| sync | Query for new messages, proposals, and commits, and send unpublished messages related to this group. |
+| find_messages | Query the database for stored messages, with filters for time frame and limit. Requires sync.|
+| list_members | A list including the account address and installation id of each member. Requires sync. |
```

Comment:
The list includes the inbox id and a list of account addresses associated with that inboxId